### PR TITLE
build 1.0.3

### PR DESCRIPTION
**Pull request by boldorider4**
* [x] checked license
* [x] updated dependencies with [upstream](https://github.com/pythological/kanren/blob/main/requirements.txt)
* [x] updated about section

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,32 +11,38 @@ source:
...
+    - python
     - setuptools
+    - wheel
   run:
-    - python >=3.6
-    - toolz
-    - cons >=0.4.0
-    - multipledispatch
-    - etuples >=0.3.1
-    - logical-unification >=0.4.1
```
Comment (rafaelmartins) [inline recipe/meta.yaml:28]: dependencies are still required: https://github.com/pythological/kanren/blob/v1.0.3/setup.py#L18-L24

Comment (boldorider4) [reply recipe/meta.yaml:28]: Thanks for catching that!

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Review by rafaelmartins — APPROVED** [review]

**Pull request merged**